src/main/scala/nodes/learning/SparseLinearMapper.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nodes.learning
+
+import breeze.linalg._
+import org.apache.spark.rdd.RDD
+import workflow.Transformer
+
+/**
+ * Computes A * x + b i.e. a linear map of data using a trained model.
+ *
+ * @param x trained model
+ * @param bOpt optional intercept to add
+ */
+case class SparseLinearMapper(
```

Comment:
So I think until we make a StandardScaler that works for both sparse and dense data, that's gonna be a little tricky to do. (Which I can do, I'm just not sure it's worth putting that in this release?)
The other thing we would be giving up is an optimized bulk apply method (unless we can make a sparse vector rows to sparse matrix util, but I'm not sure how good Breeze's support of sparse matrices is).